### PR TITLE
Reformat with Spotless

### PR DESCRIPTION
**Pull request by swankjesse**

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -2,9 +2,10 @@ root = true
...
 [*]
 indent_size = 2
+ij_continuation_indent_size = 2
 charset = utf-8
 trim_trailing_whitespace = true
 insert_final_newline = true
 
 [*.{kt, kts}]
-kotlin_imports_layout = ascii
+ij_kotlin_imports_layout = *
```
Comment (swankjesse) [inline .editorconfig:11]: I don’t know the difference between these. I’m just doing what Zipline does.

File: android-test/src/androidTest/java/okhttp/android/test/OkHttpTest.kt [hunk 2]
```diff
@@ -184,22 +189,29 @@ class OkHttpTest {
 
       var socketClass: String? = null
 
-      val clientCertificates = HandshakeCertificates.Builder()
-        .addPlatformTrustedCertificates()
-        .addInsecureHost(server.hostName)
-        .build()
+      val clientCertificates =
+        HandshakeCertificates.Builder()
```
Comment (swankjesse) [inline android-test/src/androidTest/java/okhttp/android/test/OkHttpTest.kt:193]: I don’t understand why it wrapped the constructor call?

**Comment by swankjesse** [comment]
I like the nice manual formatting better.

But I want to put robots in charge of formatting, so I’m willing to accept the robots’ dumb formatting rules.

File: android-test/src/androidTest/java/okhttp/android/test/OkHttpTest.kt (hunk 2 above)
Comment (JakeWharton) [reply android-test/src/androidTest/java/okhttp/android/test/OkHttpTest.kt:193]: Looks like it started doing https://github.com/google/google-java-format/wiki/The-Rectangle-Rule

File: mockwebserver-deprecated/src/main/kotlin/okhttp3/mockwebserver/MockResponse.kt [hunk 3]
```diff
@@ -86,62 +87,81 @@ class MockResponse : Cloneable {
...
+      headersBuilder.add(header)
+    }
 
-  fun addHeader(name: String, value: Any) = apply {
+  fun addHeader(
+    name: String,
+    value: Any,
+  ) = apply {
     headersBuilder.add(name, value.toString())
   }
```
Comment (JakeWharton) [inline mockwebserver-deprecated/src/main/kotlin/okhttp3/mockwebserver/MockResponse.kt:129]: Seems like a bug that it wraps the expression body onto its own line if the function signature is a single line, but if the function signature wraps it does not wrap the expression body.

**Review by JakeWharton — APPROVED** [review]

**Pull request merged**